pytype/typegraph/cfg.cc

```diff
@@ -504,7 +504,11 @@ PyTypeObject PyProgram = {
   tp_basicsize : sizeof(PyProgramObj),
   tp_itemsize : 0,
   tp_dealloc : ProgramDealloc,
+#if PY_VERSION_HEX >= 0x03080000 // 3.8
```

Comment:
@Solumin FYI

This change looks harmless enough to me but I'm not familiar with this code.

Reply:
Yeah, this all looks good: https://www.python.org/dev/peps/pep-0590/#changes-to-the-pytypeobject-struct

Reply:
Thanks!

Reply:
Ah yeah, FYI I explained each change in the corresponding commit message, see: https://github.com/google/pytype/pull/559/commits/cde5e9c4b0413d8123d64c0d4400efe91e8e64e5

Though I realize that's probably not the best way to communicate it in a PR...

Reply:
Ahhh I totally missed that ><